hazelcast/src/main/java/com/hazelcast/monitor/NearCacheStats.java

```diff
@@ -59,4 +59,11 @@
      * @return hit/miss ratio of the locally owned entries.
      */
     double getRatio();
+
+    /**
+     * Returns the number of evictions of locally owned entries.
```

Comment:
Right `locally owned entries` exists on other javaDocs also. But still think it is blurry and needed to fix for other methods also.

For eviction vs. expiration, adding `getExpirations` makes sense.